DEV_DOCS.md

```diff
@@ -21,6 +21,33 @@ In the future we will make this a requirement for PRs or commits.
 
 Where possible, **please rebase** your work onto master rather than merging changes from master into your PR.
 
+### Make sure tests are passing
+
+When you submit a pull request to the auspice repository, a variety of integration and unit tests will need to pass before it can be merged.
+
+You will likely want to run these tests locally before submitting:
+
+_Unit tests:_
+
+1. Install dependencies, `npm install`.
+2. Run the tests, `npm test`.
+
+_Linting:_
+
+1. Install dependencies, `npm install`.
+2. Run the linter, `npm run lint`.
+  * _If there's an issue, run `npm run lint:fix`._
+
+_Integration tests:_
+
+1. Install dependencies, `npm install`.
+2. Fetch the datasets:
+  * `npm run get-data`.
+  * `npm run get-narratives`
+3. Start the server, `npm run start`.
+4. In another terminal, run the tests:
+  * `npm run integration-test`, or...
```

Comment:
@bcoe everything here works for me locally! `integration-test` also runs the smoke tests, so it seems like you would only run `npm run smoke-test` if you just want to run the smoke tests and not the rest of the integration tests; is that a correct interpretation? If so, we could leave out 
```
, or...
* `npm run smoke-test`.
```
if we just want folks to run everything on their contributions, right?